### ISSUE TITLE
Update Cluster node properties

### ISSUE THREAD

**Issue by sarjandesai** [almende/vis#1920]
Is There any way of updating ClusterNode Properties? 

My requirement is to change the background and someother properties of the ClusterNode on click event of Network.

When i try  **update** on cluster node its converting it to normal Node. I tried to assign all cluster node related properties as well.

Sample Code :

```
var Nodes = network.body.nodes;
var xNodes = Object.keys(Nodes).map(function(k) { return Nodes[k] });// JSON to array
			
			for(var x = 0;x < xNodes.length;x++)
			{
				if(network.isCluster(xNodes[x].id))
				{
					nodes.update({ id: xNodes[x].id, label: xNodes[x].options.label,cid:xNodes[x].options.cid, shape: 'image', image: xNodes[x].options.image,color:{border: '#406897', background: '#efeded',highlight:{background:'gray',border:'blue'}}, shapeProperties: {useBorderWithImage:true}, clusterEdgeProperties : xNodes[x].clusterEdgeProperties, containedEdges : xNodes[x].containedEdges , containedNodes : xNodes[x].containedNodes, isCluster : true});
				}
				else
				{
					nodes.update({ id: xNodes[x].id, label: xNodes[x].options.label,cid:xNodes[x].options.cid, shape: 'image', image: xNodes[x].options.image,color:{border: '#406897', background: '#efeded',highlight:{background:'gray',border:'blue'}}, shapeProperties: {useBorderWithImage:true}});
				}
			}
```

**Comment by mojoaxel** [comment]
duplicate of #972

**Comment by mojoaxel** [comment]
fixed with #2055
